uui-core/src/data/processing/views/tree/BaseTree.ts

```diff
@@ -148,6 +148,11 @@ export abstract class BaseTree<TItem, TId> implements ITree<TItem, TId> {
     }
 
     public getTotalRecursiveCount() {
+        const rootInfo = this.nodeInfoById.get(undefined);
```

Comment:
I think we either should have either of:
A) Keep all totalCount-related hacks in getListProps(), and keep Tree.getTotalRecursiveCount straightforward (it should sum all counts of all leafs if all counts are known)
B) Move most logic into the Tree.getTotalRecursiveCount

I vote for A, as we have more data in getListProps, and Tree.getTotalRecursiveCount returning value bigger than we actually have in the Tree structure (in case of paging) - looks very misleading. I'd rather don't use totalCount in the Tree at all, and threat this as some generic opaque list info.